packages/style-engine/src/index.ts

```diff
@@ -62,3 +65,21 @@ export function getCSSRules(
 
 	return rules;
 }
+
+/**
+ * Returns an array of classnames.
+ *
+ * @param  style Style object.
+ *
+ * @return string[] An array of classnames.
```

Comment:
Tiny typo: we don't need to specify the return type here? (I noticed it's repeated in the generated docs)

```suggestion
 * @return An array of classnames.
```

Reply:
Thanks! I'll build the docs later 😄